components/dialogs/AuthDialog.vue

```diff
@@ -291,6 +293,11 @@ export default {
         }
       }
     },
+    currentTab(tab) {
+      if (tab === 'register') {
+        logTrackerEvent(this, 'RegFlow', 'ShowRegisterForm', 'ShowRegisterForm', 1);
```

Comment:
to prevent fire multiple times, add a local variable to store if you have fire this event before?
you clear it on `mounted` or `created` if needed